task_1/birthday_cake/main.go

```diff
@@ -12,9 +14,27 @@ package main
 */
 
 func cake(x int, y string) string {
-	panic("Implement me!")
+	var totalCandles = int32(x / 100 * 70)
```

Comment:
as far as you have more than two initial variables, let's join then into `var ( )` block